keuzewijzer-frontend/src/views/HomeView.ts

```diff
@@ -1,22 +1,47 @@
+import { StudyRoute } from '../class/StudyRoute';
+import { StudyRouteItem } from '../class/StudyRouteItem';
 import Api from '../js/api/api';
 import { View } from './View';
 
 export class HomeView implements View {
     public cohorts = Api.get('https://localhost:7298/api/Cohort');
-    public modules: any;
+    public semesterItems!: SemesterItem[];
+    public studyRouteItems: StudyRouteItem[] = [];
+
 
     public constructor() {
         console.log('HomeView.constructor()');
     }
 
-    private async getModules() {
-        const modules = await Api.get('https://localhost:7298/api/SemesterItem/cohort/2021');
-        return modules;
+    private async getSemesterItem() {
+        const semesterItems = await Api.get('https://localhost:7298/api/SemesterItem/cohort/2023');
+        return semesterItems;
+    }
+
+    private async getStudyRouteItem() {
+        const response = await Api.get('https://localhost:7298/api/StudyRoute/user/1');
+        const studyRouteItemList: StudyRouteItem[] = [];
+
+        // User not found
+        if (response.status != 404) {
+            // Tijdelijke oplossing
+            response.forEach((response: { year: number; semester: number; semesterItemId: number; }) => {
+                const year = response.year;
+                const semester = response.semester;
+                const semesterItemId = response.semesterItemId;
+
+                const studyRouteItem = new StudyRouteItem(year, semester, semesterItemId);
+                studyRouteItemList.push(studyRouteItem);
+            });
+        }

```

Comment:
Dit kan nog vanuit een viewmodel vanuit the backend.
Maar heeft geen prio voor nu denk ik